hail_search/test_utils.py

```diff
@@ -256,7 +270,16 @@
         },
     },
     'genotypeFilters': '',
-    'clinvar': None,
+    'clinvar': {
+        'alleleId': None,
+        'conflictingPathogenicities': None,
+        'goldStars': None,
+        'submitters': None,
+        'conditions': None,
+        'pathogenicity': None,
+        'assertions': None,
+        'version': '2024-02-21',
```

Comment:
I'm not sure why the entire clinvar dict was not returned in search before I changed the annotations table schema to add submitters/conditions

Reply:
Did you possibly take some of the values that were fully missing and change them to be defined but with all the subfields missing?

Reply:
this was my script to update one row: 
```
ht = ht.annotate(
    clinvar=hl.Struct(
        **dict(ht.clinvar), 
        submitters=hl.if_else(
            (ht.locus == hl.locus('chr1', 38724419, reference_genome='GRCh38')),
            ['Broad Center for Mendelian Genomics, Broad Institute of MIT and Harvard', 'Illumina Laboratory Services, Illumina', 'Blueprint Genetics','GenomeConnect, ClinGen'],
            hl.missing(hl.tarray(hl.tstr))
        ), 
        conditions=hl.if_else(
            (ht.locus == hl.locus('chr1', 38724419, reference_genome='GRCh38')),
            ['ABCA4-Related Disorders','Severe early-childhood-onset retinal dystrophy','not specified','not provided'],
            hl.missing(hl.tarray(hl.tstr))
        )
    )
```
it might be the `**dict(ht.clinvar)` that's wrong or the `hl.missing(hl.tarray(hl.tstr))`

Reply:
yeah so assigning it as a new struct is probably the issue. I would try just annotating the struct, instead of replacing it. If you annotate a missing struct it should continue to be missing
```
ht.annotate(clinvar=ht.clivar.annotate(
  submitters=hl.if_else(
            (ht.locus == hl.locus('chr1', 38724419, reference_genome='GRCh38')),
            ['Broad Center for Mendelian Genomics, Broad Institute of MIT and Harvard', 'Illumina Laboratory Services, Illumina', 'Blueprint Genetics','GenomeConnect, ClinGen'],
            hl.missing(hl.tarray(hl.tstr))
        ),
...
))
```

Reply:
thank you, this worked!